packages/flame/lib/src/text/formatters/sprite_font_text_formatter.dart

```diff
@@ -51,28 +51,28 @@ class SpriteFontTextFormatter extends TextFormatter {
   @override
   TextElement format(String text) {
     var rects = Float32List(text.length * 4);
-    var rsts = Float32List(text.length * 4);
+    var transforms = Float32List(text.length * 4);
```

Comment:
@st-pasha I have googled far and wide what rsts means but could not find it hahaha
since the parameter later becomes transforms I think that might be clearer
but if you prefer rsts I am happy to keep it if you can give some context to add to the dictionary

Reply:
Haha, etymology time!
So, I took this name from `Canvas.drawRawAtlas()`, where it has the name `rstTransforms`, which is a Float32List. But really, it is interpreted as a sequence of `RSTransform` objects. And the name `RSTransform` is derived from Skia, and presumably it means "rotation-scale transform". No idea where the extra "t" in `rstTransforms` came from, probably it was a typo.

Renaming it into simply `transforms` sounds like a good idea.

Reply:
ah, very interesting story.

so the issue is just on that flutter signature - `rsts` make sense as "rotation-scale transforms".
I imagine someone familiar with "rst" but that didn't know what it stood for called it "rst transform". similar to how people say "atm machine". funnily enough, elsewhere on flutter code it is correctly called rsTransform.
might make a PR to see what they say

as for our code, I think rsTransforms or just transforms are both good. I will merge transforms but lmk if you prefer rsTransforms I can change it